.github/scripts/generate-crd-docs/generate-crd-docs.sh

```diff
@@ -70,6 +70,13 @@ for api_group in "$OPERATOR_API_ROOT"*; do
   done
 done
 
+## Hack: sorry :(
+## Due to not adding KeptnWorkload resource into v1alpha4, there is a problem to generate CRD docs
+## in KeptnWorkloadVersion we are using KeptnWorkloadSpec from v1alpha3, which leads to using
+## ResourceReference struct from v1alpha3 -> CRD docs generator generates the docs, but it
+## refers to ResourceReference from v1alpha3, as if it is present in v1alpha4
+sed -i 's|#resourcereference|../v1alpha3/#resourcereference|' docs/content/en/docs/crd-ref/lifecycle/v1alpha4/_index.md
```

Comment:
there is a specific configuration file for the tool to exclude generation of some references, can't we double the configurations for that specific version? one that ignores  and one normal

Reply:
I did find only the option to explude it completely, but that we do not want, this is the only solution I found